fvtest/compilertriltest/ILValidatorTest.cpp

```diff
@@ -57,6 +58,69 @@ INSTANTIATE_TEST_CASE_P(ILValidatorTest, IllformedTrees, ::testing::Values(
     "(method return=Int64 (block (lreturn (sconst 1) )))"                   // lreturn incorrect type. 
     ));
 
+class InvalidPlatformVerifier : public TR::IlVerifier
+   {
+   public:
+   int32_t verifyNode(TR::Node *node)
+      {
+      return node->getOpCode().isArrayRef();
+      }
+
+   int32_t verify(TR::ResolvedMethodSymbol *sym)
+      {
+      for(TR::PreorderNodeIterator iter(sym->getFirstTreeTop(), sym->comp()); iter.currentTree(); ++iter)
+         {
+         int32_t rtn = verifyNode(iter.currentNode());
+         if(rtn)
+            return rtn;
+         }
+
+      return 0;
+      }
+   };
+
+#ifdef OMR_ENV_DATA64
+class Invalid64BitPlatform : public TRTest::JitTest, public ::testing::WithParamInterface<std::string> {};
+
+TEST_P(Invalid64BitPlatform, Fail64BitCompilation) {
```

Comment:
Does this really need to be a parameterized test? It looks like there's only one value the test parameter can have.

Reply:
Which type of test would you recommend in this case?